aws/config.go

```diff
@@ -196,6 +196,7 @@ type AWSClient struct {
 	redshiftconn          *redshift.Redshift
 	r53conn               *route53.Route53
 	partition             string
+	urlsuffix             string
```

Comment:
Can you please name this `dnsSuffix`? 

```suggestion
	dnsSuffix             string
```

Reply:
done